Makefile

```diff
@@ -63,6 +64,17 @@ start-FULL-REBUILD: echo_vars stop rm-ALL ## Remove and restart all Docker conta
 	docker-compose ${COMPOSE_FILE_ARGS} --env-file ${ENV_FILE} down
 	docker-compose ${COMPOSE_FILE_ARGS} --env-file ${ENV_FILE} up --build
 
+extract-bundles: ## Extract bundles from file-server for cloud deployment
+	/bin/rm -rf build
+	docker cp polis-${TAG}-file-server-1:/app/build/ build
+
+upload-bundles: ## upload bundles to aws s3
+	aws s3 cp build s3://${S3_BUCKET} \
```

Comment:
Will have to think on this a bit. We're planning to move away from S3 deploys, so we'll have to decide about keeping around this infrastructure. If we do accept though, would be best to explicitly name the target something like `upload-assets-to-s3`.